spcs-three-tier/ci.sh

```diff
@@ -0,0 +1,4 @@
+set -e
+bash setup.sh
```

Comment:
```suggestion
./setup.sh
```

Reply:
I had to run with `bash` because `./ ` doesn't have the feature enabled to support double square brackets. Eg.

```
if [[ "$OSTYPE" == "darwin"* ]]; then
```